pkg/constants/constants.go

```diff
@@ -21,9 +21,9 @@ const (
 	StatusResolved     = "resolved"
 
 	//ScheduleIDs to fetch Oncalls
-	ScheduleID1 = "P995J2A"
-	ScheduleID2 = "P4TU2IT"
-	ScheduleID3 = "P7CC7UN"
+	PrimaryScheduleID1 = "P995J2A"
```

Comment:
What exactly do 1, 2, 3 stand for in the IDs?

Reply:
![Screenshot from 2021-09-15 22-24-36](https://user-images.githubusercontent.com/74012736/133476610-36d91d51-fab6-4dcc-b904-1169276ea8fc.png)
I have already changed it as per your previous comment, I think you are reviewing the older code